README.md

```diff
@@ -165,61 +165,65 @@ Install Python 3
 
     sudo apt-get update
     sudo apt-get install python3
-    
+
 After the installation, please verify the version by running:
 
     python3 --version
-    
+
 It should be
-    
+
     Python 3.x.x
- 
+
 
 ### Select comm module
-We support both UDP and TCP communication (UDP is the default one).
-In the main Cmake file please set BUILD_COMM_TCP_PLAIN flag to TRUE to
-build TCP - then the test client we use will run using TCP. If you wish to
-use TCP in your application, you need to build the TCP module as mentioned
-above and then create the communication object using CommFactory
-and passing PlainTcpConfig object to it.
+We support both UDP and TCP communication. UDP is the default. In order to
+enable TCP communication, build with `-DBUILD_COMM_TCP_PLAIN=TRUE` in the cmake
+instructions shown below.  If set, the test client will run using TCP. If you
+wish to use TCP in your application, you need to build the TCP module as
+mentioned above and then create the communication object using CommFactory and
+passing PlainTcpConfig object to it.
 
 ### Build concord-bft
 
-    cd
-    cd concord-bft
-
-    # NOTE: This will add concord-bft/scripts/linux to PATH
-    #
-    . scripts/linux/set-env.sh release
-    #
-    # ...so you can invoke the make script as follows
-    #
-    make.sh
+Create a build directory and enter it:
 
+    cd
+    mkdir -p concord-bft/build
+    cd concord-bft/build
 
-For debug builds use:
+To perform a default build execute the following:
 
-    . scripts/linux/set-env.sh debug
+    cmake ..
+    make
 
+In order to turn on or off various options, you need to change your cmake configuration. This is
+done by passing arguments to cmake with a `-D` prefix: e.g. `cmake -DBUILD_TESTING=OFF`. Note that
+make must be run afterwards to build according to the configuration. The following options are
+useful for building concord-bft:
 
+ * `CMAKE_BUILD_TYPE`     - Debug | Release | RelWithDebInfo | MinSizeRel (DEFAULT Debug)
+ * `BUILD_TESTING`        - OFF | ON  (DEFAULT ON)
```

Comment:
Why off/on here, but true/false below? Is any boolean-ish string allowed (e.g. yes/no), or is something specifically checking for each value?

Reply:
Yeah, any booleanish value is supported by cmake. From the docs:


    True if the constant is 1, ON, YES, TRUE, Y, or a non-zero number. False if the constant is 0, 
    OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND. 
    Named boolean constants are case-insensitive. If the argument is not one of these constants, 
    it is treated as a variable.